test/utils/React-util.js

```diff
@@ -19,7 +19,7 @@ import {
 /**
  * Render a ReactElement and use helpful test utility methods against it.
  * @param {ReactElement} reactElement The element to render.
- * @returns {RenderedTree}
+ * @returns {RenderedTree} The return
```

Comment:
There is an eslint rule called [`valid-jsdoc`](https://github.com/eslint/eslint/blob/master/docs/rules/valid-jsdoc.md) that validates those `/**...*/` docblocks.  We don't want to require return descriptions, so I'll undo this change and add an override rule in the `test/.eslint` config to turn it off.

```json
{
  "extends": "ta-webapp/tests",
  "globals": {
    "render": false
  },
  "rules": {
    "valid-jsdoc": [2, {
      "requireReturn": false,
      "requireReturnDescription": false,
    }]
  }
}
```

The only part added there ^ is the `"rules"` object.  By specifying rules there, we are overriding the rules we are extending in the `"extends"` section (our defaults).